keras/preprocessing/sequence.py

```diff
@@ -326,9 +326,15 @@ def __init__(self, data, targets, length,
         self.reverse = reverse
         self.batch_size = batch_size
 
+        if self.start_index > self.end_index:
+            raise ValueError('`start_index+length=%i > end_index=%i` '
```

Comment:
This should read `self.start_index`, not `self.start_index+length`

Reply:
`self.start_index` is set to `start_index + length` a few lines above, so the generator never picks a index that doesn't have sufficient previous steps to fill in the memory buffer.
I could've written `self.start_index`, but I thought it would be more informative to do it like this because the users will only have access to `start_index` and `length` parameters. Should I change it anyway?